packages/js/src/settings/app.js

```diff
@@ -70,8 +71,8 @@ const Menu = ( { postTypes, taxonomies, idSuffix = "" } ) => {
 
 	return <>
 		<header className="yst-px-3 yst-mb-6 yst-space-y-6">
-			<Link to="/" className="yst-inline-block">
-				<YoastLogo className="yst-w-40" { ...svgAriaProps } />
+			<Link to="/" className="yst-inline-block yst-rounded-md focus:yst-border focus:yst-outline-none focus:yst-ring-2 focus:yst-ring-offset-2 focus:yst-ring-primary-500"  tabindex={ 0 }>
```

Comment:
* `focus:yst-border` makes it jump in the layout. What is needed for?
* `focus:yst-outline-none` seems unnecessary? Is there an outline otherwise?
* `tabindex={ 0 }` seems to enforce something which was already happening by the position on the page. I think it works the same without it?

Reply:
Hmm? I thought I could tab through it without that 🤔  You just don't see it. So I can tab to it, press enter to go to the root. So what do you mean "not focusable"?